<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StopPointWidget</class>
 <widget class="QWidget" name="StopPointWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>43</height>
   </rect>
  </property>
  <property name="whatsThis">
   <string extracomment="@info:whatsthis">A stop point in the current color range type.</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0,0,0">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string extracomment="@label:chooser The color to be used">Color:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KColorButton" name="colorChooser">
     <property name="toolTip">
      <string extracomment="@info:tooltip">Color for this stop point</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="percentageLabel">
     <property name="text">
      <string extracomment="@label:spinbox">For percentage:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="percentageSpinBox">
     <property name="toolTip">
      <string extracomment="@info:tooltip">Percentage for this stop point</string>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="addButton">
     <property name="toolTip">
      <string extracomment="@info:tooltip">Add new stop point</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Adds a new stop point after this one to the current color range type.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="removeButton">
     <property name="toolTip">
      <string extracomment="@info:tooltip">Remove this stop point</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Removes this stop point from the current color range type.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
